schematics/shell/index.ts

```diff
@@ -115,10 +116,32 @@ function addAnimationRootConfig() {
  */
 function addFontsToIndex() {
   return (host: Tree) => {
-    addHeadLink(host,
-      `<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">`);
-    addHeadLink(host,
-      `<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">`);
+    addHeadLink(host, `
```

Comment:
I generally prefer to not rely on newlines in template strings since it's not immediately obvious that they're intentional and someone might remove it

Reply:
Ya, I agree. Got a better alternative? Just `/n`?

Reply:
Yep, just `\n` is obvious that it's intentional